test/client/query-subscribe.js

```diff
@@ -592,5 +592,769 @@ module.exports = function(options) {
         });
       });
     });
+
+    describe('custom channels', function() {
```

Comment:
(Alec) Let's add a test around making sure things still work (no duplicate doc events) if multiple channels match, if debounce is disabled by the sharedb user.